tests/WikiParsing.Tests/ArticleTests.fs

```diff
@@ -93,24 +93,24 @@ let ``Gets tables``() =
     "musit"
     |> Helper.getArticle
     |> getContent
-    |> getPart "čeština"
-    |> Option.bind (getPart "sloveso")
-    |> Option.bind (getPart "časování")
-    |> Option.map getTables
-    |> Option.map (Seq.map fst >> Seq.toList)
-    |> equals (Some [ "Oznamovací způsob"; "Příčestí"; "Přechodníky" ])
+    |> (Option.get << getPart "čeština")
```

Comment:
will this just throw an exception if getPart returns None?

Reply:
Yep. Which is enough for tests I think